_topic_map.yml

```diff
@@ -21,6 +21,13 @@
 # The ordering of the records in this document determines the ordering of the
 # topic groups and topics on the main page.
 
+---
+Name: Guidines
```

Comment:
Guidelines

Reply:
The big reveal with this PR is that you are proposing that we include our docs guidelines as part of the OCP docs builds, right?

Reply:
@bobfuru, oh, yeah.

Reply:
@jboxman Couldn't we just use the GitHub "view file" option from the changes tab to view the preview? That should be exactly what it'll look like when merged.